src/components/CoverV2/Cover.tsx

```diff
@@ -166,6 +177,8 @@ const Graphic = styled.div`
 `
 
 const MetaInfo = styled.div`
+  max-width: 100%;
```

Comment:
I added this to let the `text-overflow: ellipsis` work in its children (without it the container just grew as large as it needed to be). Let me know if there is a better way.